hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -580,6 +587,20 @@ public SingleTransactionRecordBuilderImpl tokenTransferLists(
         return this;
     }
 
+    @Override
+    public TokenAirdropsRecordBuilder pendingAirdropList(@NonNull List<PendingAirdropRecord> pendingAirdropRecords) {
+        requireNonNull(tokenTransferLists, "tokenTransferLists must not be null");
+        this.pendingAirdropRecords = pendingAirdropRecords;
+        return this;
+    }
+
+    @Override
+    public TokenAirdropsRecordBuilder pendingAirdropList(@NonNull PendingAirdropRecord pendingAirdropRecord) {
```

Comment:
can be renamed to `addPendingRecord`

Reply:
I agree, the name should indicate that it adds an entry.